src/libraries/System.Text.Json/gen/Reflection/PropertyInfoWrapper.cs

```diff
@@ -31,6 +42,8 @@ public PropertyInfoWrapper(IPropertySymbol property, MetadataLoadContextInternal
 
         public override string Name => _property.Name;
 
+        public bool NeedsAtSign => _needsAtSign;
```

Comment:
nit: it doesn't require a backing field, can be just `public bool NeedsAtSign { get; }` and assigned directly in the ctor.